src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -86,6 +86,10 @@ public static int Main(string[] args)
                     return ConsoleRunner.OK;
                 }
 
+                // We already showed version as a part of the header
+                if (Options.ShowVersion)
```

Comment:
I think the if statement for WriteHeader should be `if (Options.ShowVersion || !Options.Noheader)` here and in NUnitLite. I don't think --version --noheader should display nothing, I think version should take precedence. As for @oznetmaster's comment about other command line options being ignored, I don't think showing help is a terrible sin, but if we wanted, we could move this up above help.